### PR TITLE
[Issue 717] Process page milestones section

### LINKED ISSUE #717: [Task]: Process page milestones section
### Summary

[Figma prototype](https://www.figma.com/proto/lpKPdyTyLJB5JArxhGjJnE/beta.grants.gov?type=design&node-id=918-1623&t=nFLaS0HsocPUQI8k-0&scaling=scale-down-width&page-id=918%3A1622&starting-point-node-id=918%3A1623)

image

Links: 
- API link: https://github.com/HHS/simpler-grants-gov/issues/70
- Search link: https://github.com/HHS/simpler-grants-gov/issues/89

### Acceptance criteria

- [ ] Add milestones section
- [ ] Tests pass and there is a storybook component
- [ ]  Design will need to be responsive

### PR DESCRIPTION
**Pull request by acouch**
## Summary
Fixes #717

## Changes proposed
See #717 

NOTE: The spacing seems a little off but is consistent with the other components. Suggest we refine after the content is in.

## Additional information
image

### CODE REVIEW

File: frontend/src/pages/content/ProcessMethodology.tsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+import { ExternalRoutes } from "src/constants/routes";
+import ContentLayout from "src/components/ContentLayout";
+
+const ProcessMethodology = () => {
+  const { t } = useTranslation("common", { keyPrefix: "Process" });
+
+  return (
+    <div className="padding-top-4 bg-gray-5">
+      <Grid tabletLg={{ col: 6 }} className="grid-container">
+        <strong className="tablet-lg:font-sans-lg tablet-lg:margin-bottom-05">
```
Comment (SammySteiner) [inline frontend/src/pages/content/ProcessMethodology.tsx:15]: This isn't showing up left aligned in desktop view in my browser:
image

File: frontend/src/pages/content/ProcessMethodology.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+          <Link href={ExternalRoutes.MILESTONES} passHref>
+            <Button className="margin-bottom-4" type="button" size="big">
+              {t("milestones.cta_2")}{" "}
+              <Icon.Launch
+                className="text-middle"
+                size={4}
+                aria-label="launch"
+              />
+            </Button>
+          </Link>
```
Comment (SammySteiner) [inline frontend/src/pages/content/ProcessMethodology.tsx:78]: Missing the second paragraph here
image

File: frontend/src/pages/content/ProcessMethodology.tsx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+          <p className="usa-intro">{t("milestones.paragraph_2")}</p>
+        </Grid>
+        <Grid tabletLg={{ col: 6 }}>
+          <h3 className="tablet-lg:font-sans-lg tablet-lg:margin-bottom-05">
+            {t("milestones.sub_title_3")}
+          </h3>
+          <p className="margin-top-0 font-sans-md line-height-sans-4 desktop-lg:line-height-sans-6">
+            {t("milestones.sub_paragraph_3")}
+          </p>
+          <Link href={ExternalRoutes.MILESTONES} passHref>
```
Comment (SammySteiner) [inline frontend/src/pages/content/ProcessMethodology.tsx:69]: Let's add some top margin to this button

Comment (acouch) [reply frontend/src/pages/content/ProcessMethodology.tsx:70]: This is consistent with the other buttons, let's address at in a consistent fashion in QA:

image

image

File: frontend/src/pages/content/ProcessMethodology.tsx [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+      >
+        <Grid tabletLg={{ col: 6 }}>
+          <p className="usa-intro">{t("milestones.paragraph_2")}</p>
+        </Grid>
+        <Grid tabletLg={{ col: 6 }}>
+          <h3 className="tablet-lg:font-sans-lg tablet-lg:margin-bottom-05">
+            {t("milestones.sub_title_3")}
+          </h3>
+          <p className="margin-top-0 font-sans-md line-height-sans-4 desktop-lg:line-height-sans-6">
+            {t("milestones.sub_paragraph_3")}
```
[the remark this answers is not in the document]
Comment (acouch) [reply frontend/src/pages/content/ProcessMethodology.tsx:68]: I've updated the content with the new links.

File: frontend/public/locales/en/common.json [hunk 5]
```diff
@@ -100,7 +100,23 @@
...
+      "sub_title_2": "Are you interested in the tech?",
+      "sub_paragraph_2": "We’re building a RESTful API. And we’re starting with an initial endpoint that allows API users to retrieve basic information about each funding opportunity.",
+      "cta_1": "View the API milestone on GitHub",
+      "title_2": "Milestone 2: A new search interface accessible to everyone",
+      "paragraph_2": "Once our new API is in place, we’ll begin focusing on how applicants most commonly access grants data. Our first user-facing milestone will be a simple search interface that makes data from our modern API accessible to anyone who wants to try out new ways to search for funding opportunities.",
+      "sub_title_3": "Can’t wait to try out the new search?",
+      "sub_paragraph_3": "Search will be the first feature on Simpler.Grants.gov that you’ll be able to test. It’ll be quite basic at first, and you’ll need to continue using www.grants.gov as we iterate. But your feedback will inform what happens next.",
```
[the remark this answers is not in the document]
Comment (acouch) [reply frontend/public/locales/en/common.json:116]: Fixed, thanks.

File: frontend/public/locales/en/common.json [hunk 6]
```diff
@@ -100,7 +100,23 @@
...
+      "milestone_2_goal_paragraph_2": "Be sure to sign up for product updates so you know when the new search is available.",
```
[the remark this answers is not in the document]
Comment (acouch) [reply frontend/public/locales/en/common.json]: Updated, thanks.

File: frontend/src/pages/content/ProcessMethodology.tsx (hunk 2 above)
Comment (acouch) [reply frontend/src/pages/content/ProcessMethodology.tsx:105]: I've updated with the new content.

**Comment by SammySteiner** [comment]
Please fix the failing test before merging

**Pull request merged**